panther_gazebo/include/panther_gazebo/gui/Estop.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PANTHER_GAZEBO__GUI__ESTOP_HPP_
+#define PANTHER_GAZEBO__GUI__ESTOP_HPP_
+
+#include <ignition/gui/qt.h>
+#include <ignition/gui/Plugin.hh>
+
+#include <rclcpp/rclcpp.hpp>
+#include <std_srvs/srv/trigger.hpp>
+
+#include <string>
+
+namespace ignition
+{
+namespace gui
+{
+class Estop : public Plugin
```

Comment:
```suggestion
class EStop : public Plugin
```
To make it consistent across packages.

Reply:
Honestly, there is a chance that it can't be done, I think that's what I originally did and I spent a few hours figuring out what didn't work and I discovered that two capital letters cause a problem with the QML file.
I'll check it again tomorrow.

Reply:
Ok, it works now and it probably worked when I first did it, but I didn't notice that the Gazebo plugin automatically changes its name to `E Stop` and I didn't find it.
![image](https://github.com/user-attachments/assets/474e9493-0110-424d-8c37-6df8d99a932d)